src/collections/_documentation/platforms/node/pluggable-integrations.md

```diff
@@ -51,10 +44,10 @@ Available options:
 
 _Import name: `Sentry.Integrations.Modules`_
 
-This integration fetches names of all currently installed node modules and attaches the list to the event. Once fetched, the list will be cached for later reuse.
+This integration fetches names of all currently installed Node modules and attaches the list to the event. Once fetched, the list will cache for later reuse.
```

Comment:
List itself is the thing that will be stored (cached) and it's not a thing that stores (caches) things. Should it be `cache` and not `be cached` for sure?

Reply:
Hmmm... the sentence is in passive voice because it's missing a subject. What if it reads, "Once fetched, Sentry will cache the list for later use." This way, it's clear what is doing the caching. Am I correct that Sentry caches the list? Or is something else performing the cache?